circle.yml

```diff
@@ -3,17 +3,24 @@ dependencies:
     - ~/.pyenv/versions/3.4.3
     - ~/coala-atom/node_modules
   pre:
-    # Install atom dependencies
+    # Install  dependencies
+    - sudo apt-get install libnotify4
     - sudo apt-get update
     - sudo apt-get install gvfs-bin
     # Install atom
     - curl -L -o atom.deb https://atom.io/download/deb
     - sudo dpkg --install atom.deb
-
+    - sudo apt-get -f install
     # Set python version
     - pyenv local 3.4.3
 
+    # install NLTK
+    - pip3 install -U nltk
+    # install NLTK data
+    - python3 -m nltk.downloader punkt maxent_treebank_pos_tagger averaged_perceptron_tagger
```

Comment:
We won't need this after https://github.com/coala/coala-bears/pull/1619#pullrequestreview-33432670; it'll need to be added to `cache_directories` though.